api/src/main.ts

```diff
@@ -6,14 +6,15 @@ import { SwaggerModule } from "@nestjs/swagger";
 import swaggerConfig, { swaggerOptions } from "swagger/swagger.config";
 import { JwtToken } from "swagger/jwt-token.property.dto";
 import { User } from "users/model/users.entity";
+import {Message} from "messages/model/message.entity";
+
+const IS_DEVELOPMENT = process.env.NODE_ENV === "dev";
+const VALIDATION_OFF = process.env.VALIDATION_OFF === "true";
+const ENABLE_ALL_CORS = process.env.ENABLE_ALL_CORS === "true";
```

Comment:
Stick to one naming convention.

```suggestion
const IS_DEVELOPMENT_ENABLED = process.env.NODE_ENV === "dev";
const IS_VALIDATION_ENABLED = process.env.VALIDATION_OFF === "true";
const IS_EACH_CORS_ENABLED = process.env.ENABLE_ALL_CORS === "true";
```

Reply:
I will do it.

Reply:
DONE